### DISCUSSION TITLE
Recoverycode not working

### DISCUSSION THREAD

**Discussion by FlashstaRR** [dani-garcia/vaultwarden#2439] [Q&A]
Hey, 

if i set up TOTP with the Google Authenticator everything works fine. As i saw at the user Settings (/#/settings/two-factor) i can display (and safe) my backupcode to recover if i will lost my Phone or something. The Backupcode is 32 digits like "ABCD EFGH IJKL...". 

I wanted to test this Backupcode so i logged in with my Password and came to the screen which says "please enter the 6 digits" under this there is a link to "use a different 2FA method". If i click it a pop-up will show which shows the "TOTP" and "Recoverycode" if i click on "Recovercode" to put my code in there it only links to "https://bitwarden.com/help/lost-two-step-device/" so where do i put my code in? 

I tried to put the 32 digit code in the 6 digits field but then the error massage is "TOTP is not a number" 

Im using Version 2.25.1

Please excuse my translation but my vaultwarden is running in german. I hope you still get it ;)

**Comment by jjlin** [comment]
https://bitwarden.com/help/lost-two-step-device/

links to:

https://bitwarden.com/help/two-step-recovery-code/#use-your-recovery-code

which describes where to input the recovery code. It does seem more confusing than it needs to be...
